### DISCUSSION TITLE
Display multiple values in Search Result

### DISCUSSION THREAD

**Discussion by CalvinHew** [microsoft-search/pnp-modern-search#2762] [General]
I'm using Person column which allows multiple selections. I would like to show multiple display names, separated by comma in the search result webpart. I am unable to do so in both the html and adaptive card. Anyone has done that before? Thanks.

**Comment by wobba** [comment]
First map the columns crawled property to a managed property (unless you use site columns). Then add the property to be retrieved in the web part. Use e.g. the debug template to look at the data format of what is returned. Then add appropriate string manipulation to the handlebar template to have it displayed as you want.

Here's a guide to get you started on property mapping https://learn.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search

**Reply by CalvinHew** [reply]
Thanks
